docs/private-link.md

```diff
@@ -95,6 +95,8 @@ For workspace default ACR, please enable firewall exception on allow trusted Mic
 More documentation for a private link container registry is provided here: https://docs.microsoft.com/en-us/azure/container-registry/container-registry-private-link#set-up-private-endpoint---portal-recommended.
 Configure compute cluster for building image in your vnet. https://docs.microsoft.com/en-us/azure/machine-learning/how-to-secure-workspace-vnet?tabs=pe#enable-azure-container-registry-acr
 
+Please note the admin user of your ARC should be enabled. Trainging job is using this crediental to access your ACR. If you don't want to enable admin user, a workaround can be found in this document: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-identity-based-service-authentication?tabs=cli#scenario-azure-container-registry-without-admin-user
+
```

Comment:
this doc dosen't mention any thing regarding to k8s compute, actully for our amlarc compute, no matter use system or user identity, customers should manually assign the ACRPull role to compute.